### ISSUE TITLE
Paper 1.20.4 - Custom Skin for humanoid entities

### ISSUE THREAD

**Issue by CallmeDome** [kangarko/Boss#1237]
Good Day! I'm considering buying your plugin, but I'm not sure yet, if your plugin contains the features I need. I hope to get some answers here. Is it possible to create a humanoid bosses/entities and give or change their skin? Somehow like the citizens plugin? Is it possible to change the appearance of the pillager NPCs, which spawns naturally, to a humanoid entity?

**Comment by kangarko** [comment]
Yes, using Citizens NPC plugin. 

Yes you can also setup Boss to replace vanilla mobs spawning.

**Comment by CallmeDome** [comment]
> Yes, using Citizens NPC plugin.
> 
> Yes you can also setup Boss to replace vanilla mobs spawning.

How do you mean using Citizens NPC Plugin? Is it possible to create human npcs and give them custom skins by providing a download link or searching the right mc Player skin?

**Comment by kangarko** [comment]
If you install the Citizens plugin then Boss plugin will connect to it and use it to spawn human NPCs with custom skins. If Citizens supports downloading links or searching skins online then Boss will support it.

**Comment by CallmeDome** [comment]
Good Day, I open this thread again, because my question fits in here pretty well. I have bought and installed the boss and I'm very happy with it so far. 
Now where I'm trying to set my npcs up, I coming to a little problem. I would like to create an citizen npc, which spawns naturally instead of an vanilla monster. Sadly I wasn't able to find a way to do that. 
Either I was able to create a player npc the way I like it, but wasn't able to get him spawning instead on an vanilla monster, or I created a monster npc, but wasn't able to change his apperance to an citizen npc.

Is there a way how I can solve that?
